config/locales/simple_form.en.yml

```diff
@@ -53,6 +53,7 @@ en:
       profile:
         address: "Street Address for Pickup"
         notes: "Pickup Notes"
+        notes: "Phone Number"
```

Comment:
@corwinharrell how should we label this input?

Is `Phone Number` enough, or should we be more explicit -- something like `Phone number to contact about donations`?